kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/DeltaTableWritesSuite.scala

```diff
@@ -166,6 +167,59 @@ class DeltaTableWritesSuite extends DeltaTableWriteSuiteBase with ParquetSuiteBa
     }
   }
 
+  test("create table and set properties") {
+    withTempDirAndEngine { (tablePath, engine) =>
+      val table = Table.forPath(engine, tablePath)
+      val txnBuilder1 = table.createTransactionBuilder(engine, testEngineInfo, CREATE_TABLE)
+
+      val txn1 = txnBuilder1
+        .withSchema(engine, testSchema)
+        .build(engine)
+
+      txn1.commit(engine, emptyIterable())
+
+      val ver0Snapshot = table.getSnapshotAsOfVersion(engine, 0).asInstanceOf[SnapshotImpl]
+      assert(TableConfig.CHECKPOINT_INTERVAL.fromMetadata(ver0Snapshot.getMetadata) == 10)
+
+      val txnBuilder2 = table.createTransactionBuilder(engine, testEngineInfo, WRITE)
+      val txn2 = txnBuilder2
+        .withTableProperties(engine, Map(TableConfig.CHECKPOINT_INTERVAL.getKey -> "2").asJava)
+        .build(engine)
+
+      txn2.commit(engine, emptyIterable())
+
+      val ver1Snapshot = table.getSnapshotAsOfVersion(engine, 1).asInstanceOf[SnapshotImpl]
+      assert(TableConfig.CHECKPOINT_INTERVAL.fromMetadata(ver1Snapshot.getMetadata) == 2)
+    }
+  }
+
+  test("create table - invalid properties - expect failure") {
+    withTempDirAndEngine { (tablePath, engine) =>
+      val table = Table.forPath(engine, tablePath)
+      val txnBuilder = table.createTransactionBuilder(engine, testEngineInfo, CREATE_TABLE)
+
+      val ex1 = intercept[UnknownConfigurationKeyException] {
+        txnBuilder
+          .withSchema(engine, testSchema)
+          .withTableProperties(engine, Map("invalid key" -> "10").asJava)
+          .build(engine)
```

Comment:
make an utility method for this code block and use it it across tests. Simplifies the test code.

something like `Transaction createTxn(engine, testSchema, properties)`

Reply:
If there is already an existing method, add the param for config.